### PR TITLE
Handling Enterprise Architect 15 exports

### PR DESCRIPTION
**Pull request by kwkwan**

### CODE REVIEW

File: spec/lutaml/parsers/xmi_251_spec.rb [hunk 1]
```diff
@@ -262,17 +253,17 @@
...
               },
               {
                 definition: nil,
                 name: "recommendation",
-                type: "extension_Bibliography",
+                type: nil,
               },
               {
                 definition: nil,
                 name: "premission",
```
Comment (ronaldtse) [inline spec/lutaml/parsers/xmi_251_spec.rb:265]: This is "permission"?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: spec/lutaml/parsers/xmi_251_spec.rb (hunk 1 above)
Comment (kwkwan) [reply spec/lutaml/parsers/xmi_251_spec.rb:265]: This value is from line 61 of the file`spec/fixtures/ea-xmi-2.5.1.xmi`.

```xml
<ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="EAID_547E848A_0513_4565_8B45_495BC23F9B3D" name="premission"/>
```

Comment (ronaldtse) [reply spec/lutaml/parsers/xmi_251_spec.rb:265]: Got it. Can you change both to the correct spelling? Thanks.

Comment (kwkwan) [reply spec/lutaml/parsers/xmi_251_spec.rb]: OK.  I will correct the spelling.

**Pull request merged**